src/SFA.DAS.Campaign.Web/Views/Shared/Components/FeedbackBanner/_feedbackBanner.cshtml

```diff
@@ -0,0 +1,13 @@
+﻿<div class="feedback-banner" data-module="alertBanner" data-cookieName="FeedbackBanner">
+    <div class="feedback-banner--inner">
+        <p class="feedback-banner__content feedback-banner__content--text" >
+            <span class="badge badge--apprentice">
+                Feedback
+            </span> This is a new service – your <a class="feedback-banner__link" href="https://www.smartsurvey.co.uk/s/S2NQQ">feedback</a> will help us to improve it.
```

Comment:
I think the content should say `This is a free service – your feedback will help us to improve it.` You've said 'new' service